pioneer_control/Vec2i32 origin
pioneer_control/Vec2i32 goal
---
int32 distance
